### ISSUE TITLE
Check if an integer is greater than some value 

### ISSUE THREAD

**Issue by icaroharry** [ctavan/express-validator#428]
Hello!

When dealing with floats we can do:
`req.checkBody('var', 'var must be float and greater than 0.0').isFloat({ gt: 0.0 });`

Is there any particular reason why it is not possible to do this with `isInt()`?

Thanks

**Comment by gustavohenke** [comment]
Hello @icaroharry!
I couldn't reproduce your issue. Can you show an example request that fails?

I'm suspecting that if `isFloat` works for you, the middleware is receiving float values.

**Comment by icaroharry** [comment]
Sorry. I think I wasn't very clear about my issue.

Also, I just realized that the library already supports what I was requesting. I wanted to do a similar checking as the example I gave above, but instead of using `isFloat()`, I wanted to use `isInt()`. Something like:

`req.checkBody('var', 'var must be an integer greater than 3').isFloat({ gt: 3 });`

For some reason it seemed to me that it wasn't possible to do this with the library, also I didn't see [this](https://github.com/chriso/validator.js#validators) in the docs. So I realized that what I said makes no sense :P 

Sorry about that.
